### DISCUSSION TITLE
Import inside Pictures (pictures/import)

### DISCUSSION THREAD

**Discussion by plant7403** [photoprism/photoprism#2469] [Q&A]
Hi, I want my hard drive/home directory be clean
So I have only few directories like Pictures, Music, Documents, etc
I want to put Import inside Pictures directory
How to do that? Thank you

**Comment by lastzero** [comment]
You can skip mounting an import folder in this case. Import moves files to the pictures / originals folder. It must therefore not be inside since this creates a loop which can have unwanted consequences.
